rio_tiler/colormap.py

```diff
@@ -138,11 +137,13 @@ class ColorMaps:
     """Default Colormaps holder.
 
     Args:
-        data (dict): colormaps.
+        data (dict): colormaps. Defaults to `rio_tiler.colormap.DEFAULTS_CMAPS`.
 
     """
 
-    data: Dict[str, Union[str, numpy.array]] = attr.ib()
+    data: Dict[str, Union[str, numpy.array]] = attr.ib(
+        default=attr.Factory(lambda: DEFAULTS_CMAPS)
+    )
```

Comment:
thanks @geospatial-jeff for this ;-)

Reply:
Tag: non-doc change but I couldn't resist

Reply:
I'd suggest `DEFAULT_CMAPS` and `DEFAULT_CMAPS_FILES`